core/src/com/mygdx/battleship/BattleshipBots/RandomBot.java

```diff
@@ -0,0 +1,146 @@
+package com.mygdx.battleship.BattleshipBots;
+
+import java.util.*;
+
+import com.mygdx.battleship.BattleshipUtils;
+import com.mygdx.battleship.MoveResult;
+
+
+public class RandomBot extends BattleshipBot  {
+
+    private final List<String> moves = new ArrayList<>(100);
+
+    /**
+     * Constructor
+     */
+    public RandomBot(String name) {
+        super(name);
+        setupMoves();
+    }
+
+    private void setupMoves() {
+        for (int i = 0; i < 10; ++i){
+            for (int j = 1; j <= 10; ++j){
+                char row = (char) (i + 'A');
+                moves.add("" + row + j);
+            }
+        }
+        Collections.shuffle(moves);
+    }
+
+    /**
+     * Decide ship placement.
+     * Sets ship placement variable.
+     */
+    private void shipPlacements() {
+        //start from the largest ship.
+        int curShip = 0;
+
+        HashSet<String> usedCoordinates = new HashSet<>();
+
+        while (curShip < 5) {
+
+            if(usedCoordinates.size() > 2+3+3+4+5){
+                int y = 2;
+            }
+
+            int orientation;
+            boolean reset = false;
+            String[] placement = new String[shipSizes[curShip]];
+            //decide random starting position and orientation
+            Random rand1 = new Random();
+            Random rand2 = new Random();
+            char startL = (char) (rand1.nextInt(10) + 'a');
+            int  startR = rand1.nextInt(10);
```

Comment:
nextInt(int n)
          Returns a pseudorandom, uniformly distributed int value between 0 (inclusive) and the specified value (exclusive), drawn from this random number generator's sequence.

Probably need to add 1 to startR